conf/spark-defaults.conf.template

```diff
@@ -19,9 +19,16 @@
 # This is useful for setting default environmental settings.
 
 # Example:
-# spark.master                     spark://master:7077
-# spark.eventLog.enabled           true
-# spark.eventLog.dir               hdfs://namenode:8021/directory
-# spark.serializer                 org.apache.spark.serializer.KryoSerializer
-# spark.driver.memory              5g
-# spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
+# spark.master                          spark://master:7077
+# spark.eventLog.enabled                true
+# spark.eventLog.dir                    hdfs://namenode:8021/directory
+# spark.serializer                      org.apache.spark.serializer.KryoSerializer
+# spark.driver.memory                   5g
+# spark.executor.extraJavaOptions       -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
+
+#spark.ui.allowFramingFrom              https://example.com/
+#spark.ui.xXssProtection.enabled        1; mode=block
```

Comment:
This sounds like a boolean flag but its value isn't boolean; if it has only one reasonable value then it can be a boolean that sets that value. Otherwise maybe call the property something else

Reply:
Possible values can be one from the below set, default one mentioned in template is commonly-used and provides effective protection.
X-XSS-Protection: { 0 | 1 |1; mode=block | 1; report=<reporting-uri> }

Reply:
I would remove '.enabled' from the name then

Reply:
@srowen renamed the keys as suggested. Thanks again for the review.